frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/StartService.java

```diff
@@ -183,10 +185,19 @@ public UserStartInfo buildUserStartInfo(
 
         boolean isBlockedForReauth = false;
         if (configurationService.isAuthenticationAttemptsServiceEnabled()) {
-            isBlockedForReauth =
+            var reauthCountTypesToCounts =
```

Comment:
Should this be singular (`reauthCountTypesToCount`)?  Could it be renamed to `errorTypesWithCounts`?

Reply:
As discussed - will have a think about this as part of next PR 👍